libs/ssl-config/src/test/java/org/elasticsearch/common/ssl/SslConfigurationLoaderTests.java

```diff
@@ -103,6 +103,7 @@ public void testLoadTrustFromPemCAs() {
     }
 
     public void testLoadTrustFromPkcs12() {
+        assumeFalse("Can't use JKS/PKCS12 keystores in a FIPS JVM", inFipsJvm());
```

Comment:
It may be outside of the scope of this PR, but do we plan to test Fips compatible PKCS12 keystores?

Reply:
There is no such thing as Fips compatible PKCS12 keystores, unfortunately*. See https://downloads.bouncycastle.org/fips-java/BC-FJA-UserGuide-1.0.2.pdf section 7 too. We document this in https://www.elastic.co/guide/en/elasticsearch/reference/current/fips-140-compliance.html#_tls_keystores_and_keys

\* Technically, I think it might be possible to create a PKCS12 implementation that uses only PBES2 ( i.e. PBKDF2 ) for key derivation, but that would be outside the scope of this PR and our FIPS 140 support too.

Reply:
Aha, I must have mistaken FIPS compatible with FIPS approved only compatible...

Reply:
Not sure what you mean by compatible here, we can also discuss live